### DISCUSSION TITLE
[Suggestion] 在文档中添加指示，以终结有关 Linux 和其他平台支持的询问

### DISCUSSION THREAD

**Discussion by w568w** [HMBSbige/NatTypeTester#233] [Ideas]
我看到有一些关于 Linux 和其他平台版本（#230, #192, #184），以及关于命令行工具支持的请求（#217）。

考虑到这部分高级用户，或许可以在 README 中添加这些平台上的类似软件的链接，引导他们去使用其他工具，以避免重复提问？

一个好的例子是 [STUNTMAN](https://www.stunprotocol.org/)，实现了 RFC 3489、5389、5769 和 5780，支持 Unix 平台：

```
$ stunclient --mode full stun.miwifi.com
Binding test: success
Local address: x.x.x.x:36360
Mapped address: y.y.y.y:45368
Behavior test: success
Nat behavior: Endpoint Independent Mapping
Filtering test: success
Nat filtering: Address and Port Dependent Filtering
```

**Comment by HMBSbige** [comment]
怎么说呢？Linux 是会支持的
而且其他项目也跟本 repo 无关

**Reply by w568w** [reply]
这样啊，那我会错意了。从你在 #192 的回复来看，我以为本项目不打算支持其他平台…

**Reply by HMBSbige** [reply]
那是两年前...
